app/views/external_users/claims/misc_fees/advocates/supplementary/_misc_fee_fields.html.haml

```diff
@@ -1,57 +1,64 @@
 - fee = present(f.object)
-.multiple-choice{"data-target" => to_slug(f.object.description), "class" => "fx-hook-#{fee.fee_type_code.downcase}"}
+.app-checkboxes__item{ 'data-target': to_slug(f.object.description), class: "fx-hook-#{fee.fee_type_code.downcase}" }
+  = f.govuk_check_box :toggle,
+    1,
+    checked: !f.object.blank?,
+    class: 'fx-checkbox-hook',
+    label: { text: f.object.description },
+    'data-unique-code': f.object.fee_type.unique_code do
 
-  = f.check_box(:toggle, checked: !f.object.blank?, class: "fx-checkbox-hook", id: "#{to_slug(f.object.description)}-input", "aria-controls" => to_slug(f.object.description), :name => "misc_fees_checklist_#{f.object.fee_type_code.downcase}", data: { unique_code: f.object.fee_type.unique_code})
+    .misc-fee-group.fx-fee-group.js-block.fx-do-init{ data:{type:"miscFees", autovat: @claim.apply_vat? ? "true" : "false", "block-type": "FeeBlockCalculator"} }
+      = f.govuk_text_field :fee_type_id,
+        class: 'js-fee-calculator-fee-type js-fee-type',
+        form_group: {classes: 'govuk-visually-hidden'},
+        label: { hidden: true },
+        type: 'hidden'
 
-  %label{:for => "#{to_slug(f.object.description)}-input"}
-    = f.object.description
+      - if f.object.fee_type.unique_code.eql?('MIUMO')
+        = render 'warnings/unused_material_over_3_hours'
 
-  = f.hidden_field :_destroy, class: 'destroy', value: f.object.blank?
+      = f.govuk_number_field :quantity,
+        class: 'quantity fee-quantity js-fee-quantity js-fee-calculator-quantity',
+        form_group: { classes: 'quantity_wrapper' },
+        hint: { text: t('.quantity_hint') },
+        label: { text: t('.quantity_html', context: f.object.description) },
+        min: 1,
+        value: fee.quantity,
+        width: 3
 
-.panel.panel-border-narrow.js-hidden{:id => to_slug(f.object.description)}
-  .misc-fee-group.fx-fee-group.js-block.fx-do-init{data:{type:"miscFees", autovat: @claim.apply_vat? ? "true" : "false", "block-type": "FeeBlockCalculator"}}
-    = f.hidden_field :fee_type_id, class: 'js-fee-calculator-fee-type js-fee-type'
+      = f.govuk_number_field :rate,
+        class:'rate fee-rate js-fee-calculator-rate',
+        form_group: {classes: 'js-unit-price-effectee calculated-unit-fee'},
```

Comment:
Nit: space before and after braces

TODO: implement haml linting

Reply:
Done